sdk/storage/azure-storage-blob-changefeed/README.md

```diff
@@ -0,0 +1,190 @@
+
+
+## Getting started
+
+### Prerequisites
+* Python 2.7, or 3.5 or later is required to use this package.
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) and an
+[Azure storage account](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account) to use this package.
+
+### Install the package
+Install the Azure Storage Blob ChangeFeed client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-storage-blob-changefeed --pre
+```
+
+### Create a storage account
+If you wish to create a new storage account, you can use the
+[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-the-azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-powershell),
+or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-azure-cli):
+
+```bash
+# Create a new resource group to hold the storage account -
+# if using an existing resource group, skip this step
+az group create --name my-resource-group --location westus2
+
+# Create the storage account
+az storage account create -n my-storage-account-name -g my-resource-group --hierarchical-namespace true
+```
+
+### Authenticate the client
+
+Interaction with Blob ChangeFeed client starts with an instance of the ChangeFeed class. You need an existing storage account, its URL, and a credential to instantiate the client object.
+
+#### Get credentials
+
+To authenticate the client you have a few options:
+1. Use a SAS token string 
+2. Use an account shared access key
+3. Use a token credential from [azure.identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity)
+
+Alternatively, you can authenticate with a storage connection string using the `from_connection_string` method. See example: [Client creation with a connection string](#client-creation-with-a-connection-string).
+
+You can omit the credential if your account URL already has a SAS token.
+
+#### Create client
+
+Once you have your account URL and credentials ready, you can create the ChangeFeedClient:
+
+```python
+from azure.storage.blob.changefeed import ChangeFeedClient
+
+service = ChangeFeedClient(account_url="https://<my-storage-account-name>.dfs.core.windows.net/", credential=credential)
+```
+
+## Key concepts
+
+#### Clients
+
+The Blob ChangeFeed SDK provides one client:
+ChangeFeedClient: get change feed events by page, get all change feed events, get events in a time range, start listing events with a continuation token
```

Comment:
This section isn't rendering quite right - I think it needs a bullet point:
https://github.com/Azure/azure-sdk-for-python/blob/bae3f4aefd644fb974e016ab3fb355b19ad15316/sdk/storage/azure-storage-blob-changefeed/README.md#clients

Also description needs a capital and a final period.